annotation/util.scm

```diff
@@ -388,7 +388,7 @@
       (run-query
         (BindLink
           (VariableNode "$loc")
-          (AndLink
+          (ContextLink
             (MemberLink 
               child
               parent)
```

Comment:
On the other hand, if no such MemberLink existed before, adding it this way creates it. But if that is what you want, it can just be added directly, independent of the BindLink